app/agents/voice/automatic/tools/breeze/utils.py

```
@@ -193,19 +193,28 @@ async def patch_shop_config(
 
 
 def format_announcement_html(
-    description: str, background_color: str = "#714acd"
+    description: str, 
+    background_color: str = None,
```

Ruff (0.13.3)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)